src/m365/outlook/commands/message/message-list.ts

```diff
@@ -67,19 +137,40 @@ class OutlookMessageListCommand extends GraphCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
-      const folderId = await this.getFolderId(args);
+      if (!args.options.userId && !args.options.userName && accessToken.isAppOnlyAccessToken(auth.connection.accessTokens[this.resource].accessToken)) {
+        throw 'You must specify either the userId or userName option when using an app-only access token';
+      }
+
+      const userUrl = args.options.userId || args.options.userName ? `users/${args.options.userId || args.options.userName}` : 'me';
+
+      const folderId = await this.getFolderId(userUrl, args);
+      const folderUrl: string = folderId ? `/mailFolders/${folderId}/messages` : '/messages';
+      let requestUrl = `${this.resource}/v1.0/${userUrl}${folderUrl}`;
```

Comment:
The default page size for outlook messages is 10, this is quite low imo. Maybe let's increase it to `100` which is most of the time the default page size?